releng/buildpacks/auditlog-client/src/main/java/com/sap/auditlog/client/deserializers/InstantDeserializer.java

```diff
@@ -0,0 +1,15 @@
+package com.sap.auditlog.client.deserializers;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import java.io.IOException;
+import java.time.Instant;
+
+public class InstantDeserializer extends JsonDeserializer<Instant> {
+
+  public Instant deserialize(JsonParser p, DeserializationContext ctxt) throws IOException {
```

Comment:
where is ctxt used ?

Reply:
Unfortunately `ctxt` is part of the interface `JsonDeserializer<..>` and it isn't used in the implementation. I'll try to find another way to deserialize it. Jackson doesn't support natively the `Instant` type, which is unfortunate.